src/main/flight/pid_init.c

```diff
@@ -272,6 +274,8 @@ void pidUpdateFeedforwardLpf(uint16_t filterCutoff)
     if (filterCutoff > 0) {
         for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
             pt3FilterUpdateCutoff(&pidRuntime.feedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
+            pt3FilterUpdateCutoff(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
+            pt2FilterUpdateCutoff(&pidRuntime.angleFeedforwardPt2[axis], pidRuntime.feedforwardSmoothFactor);
```

Comment:
Further testing strongly suggests that we should smooth `angleFeedforward` steps much harder than we currently do.  I have been testing `pidRuntime.feedforwardSmoothFactor * 0.1f` with good results at 500hz.  That means a value around 0.03 for the filter gain.

The additional delay is of no consequence, and the smoother signal is very helpful.

Since we don't know for sure that a user will configure the feedforward smooth factor correctly, this cutoff could be set in relation to the link time.  For instance, at 500hz, time interval is 0.002, so we could use the link interval * 15.  We would get 0.03, which would be nice and smooth.

When the feedforward smoothing is very strong, we can apply a lot more without inducing wobbles, and the motor traces are much smoother.

Reply:
changed.